runtime/fixtures/simple_main.dart

```diff
@@ -26,6 +27,18 @@ void canRegisterNativeCallback() async {
 
 void NotifyNative() native "NotifyNative";
 
-
 @pragma('vm:entry-point')
 void testIsolateShutdown() {  }
+
+@pragma('vm:entry-point')
+void testCanSaveCompilationTrace() {
+  List<int> trace = null;
+  try {
+    trace = saveCompilationTrace();
+  } catch (exception) {
+    print("Could not save compilation trace: " + exception);
+  }
+  NotifyResult(trace != null && trace.length > 0);
```

Comment:
Here is the test that I mentioned that does more thorough check on the output: https://github.com/flutter/flutter/blob/d2790bd2bbf1851db2359026df7bca38e255b9d3/packages/flutter/test/foundation/service_extensions_test.dart#L655